soc/arm/nuvoton_npcx/npcx7/mpu_regions.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "../../common/cortex_m/arm_mpu_mem_cfg.h"
+
+static const struct arm_mpu_region mpu_regions[] = {
+	MPU_REGION_ENTRY("FLASH_A",
```

Comment:
Is it better to use "FLASH0_0"/"FLASH0_1" or "FLASH0_A"/"FLASH0_B" instead of "FLASHA" and "FLASHB" since the node name is "[flash0](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/nuvoton/npcx7m7fc.dtsi#L11)"?

Reply:
Yeah good idea, default is FLASH_0 and SRAM_0, changing to FLASH_0_0, FLASH_0_1 and SRAM_0 as well for consistency.